.travis.yml

```diff
@@ -12,9 +12,9 @@ branches:
 matrix:
   include:
     - os: linux
-      dist: trusty
+      dist: xenial 
       sudo: required
-      env: SWIFT_SNAPSHOT=4.0.3
+      env: DOCKER_IMAGE=swift:4.0.3 SWIFT_SNAPSHOT=4.0.3
```

Comment:
Just spotted this - we intentionally left a build running against Trusty (14.04) since people may conceivably still be running this. Let's revert this so we retain the coverage.